gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/resources/scripts/create-index.js

```diff
@@ -34,6 +34,7 @@ db.getCollection(`${prefix}plans`).reIndex();
 db.getCollection(`${prefix}subscriptions`).dropIndexes();
 db.getCollection(`${prefix}subscriptions`).createIndex( { "plan" : 1 }, { "name": "p1" } );
 db.getCollection(`${prefix}subscriptions`).createIndex( { "application" : 1 }, { "name": "a1" } );
+db.getCollection(`${prefix}subscriptions`).createIndex( { "createdAt" : -1 }, { "name": "c-1" } );
```

Comment:
should not it be `c1` ? to follow other indices convention ?

Reply:
1 means ascending, -1 means descending
so for me the name is accurate.

Maybe other indices are not well written ;)

Reply:
oups, did not understood like that :) thanks